src/plugins/intel_gpu/tests/unit/passes/test_module_fusing_reorder.cpp

```diff
@@ -297,9 +297,9 @@ TEST_P(test_can_fuse_reorder_onednn, reorder_for_firstconv_onednn)
 
 INSTANTIATE_TEST_SUITE_P(testing_can_fuse_reorder_first_conv, test_can_fuse_reorder_onednn,
                         ::testing::ValuesIn(std::vector<reorder_test_param>{
-                                            reorder_test_param{format::bs_fs_yx_bsv8_fsv4, format::b_fs_yx_fsv32, data_types::f32, data_types::u8, {1, 3, 8, 8}, {1, 32, 8, 8}, {1, 3, 1, 1},
```

Comment:
Why do we need to change this?

Reply:
this test call  init_graph() in program instantiation(because of no_optimizations switch), calculation of shapes was added to this step(test not done it before) . In convolution is method calc_output_layout_impl: which has line weights_layout = weights_layout.convert_to_weights_layout(desc->grouped_weights_shape);  (https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_gpu/src/graph/convolution.cpp) and here https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_gpu/src/runtime/layout.cpp?plain=1#L128 weights must have specific format

Reply:
Got it
However, it looks like the test has some typos:
1) Currently weights use `p.input_format`, but not `p.weights_format` (that is why you needed to make this change). It should probably be like this instead:
`auto weights = engine.allocate_memory({ p.input_data_type, p.weights_format, p.weight_shape });`
2) The weights format is specified as `format::goiyx`, but the actual convolution primitive is created without groups: `cldnn::convolution("conv", input_info("reorder_input"), "weights", "", 1, {1, 1}, {1, 1}, {0, 0}, {0, 0}, false)`
So, the test parameters should be as follows (keep the original input format, but with fixed weights):
```
                                            reorder_test_param{format::bs_fs_yx_bsv8_fsv4, format::b_fs_yx_fsv32, data_types::f32, data_types::u8, {1, 3, 8, 8}, {1, 32, 8, 8}, {1, 3, 1, 1},
                                                tensor{1}, tensor{0}, data_types::u8, format::oiyx, true},
```

Reply:
done